apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -308,6 +316,19 @@ export const SignedVerdictOverview: React.FC = () => {
                 <PoliceRequestAccordionItem workingCase={workingCase} />
                 <CourtRecordAccordionItem workingCase={workingCase} />
                 <RulingAccordionItem workingCase={workingCase} />
+                {workingCase.files && workingCase.files.length > 0 && (
+                  <AccordionItem
+                    id="id_4"
+                    label={`Rannsóknargögn (${workingCase.files.length})`}
+                    labelVariant="h3"
+                  >
+                    <CaseFileList
+                      caseId={workingCase.id}
+                      files={workingCase.files}
+                      canOpenFiles={false}
```

Comment:
Shouldn't prosecutors be able to open case files of completed cases?
Later, registrars will be able to open case files of completed appealed cases.